src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrganizationContext.tsx

```diff
@@ -0,0 +1,309 @@
+//     priority: 1,
+//     providerId: 'AWS',
+//     providerName: 'Amazon Web Services',
+//     regionId: 'us-east-1',
+//     regionName: 'US East (N. Virginia)',
+//   },
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: false,
+//     priority: 2,
+//     providerId: 'AWS',
+//     providerName: 'Amazon Web Services',
+//     regionId: 'us-east-1-2',
+//     regionName: 'US East (N. Virginia) 2',
+//   },
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: false,
+//     priority: 1,
+//     providerId: 'AWS',
+//     providerName: 'Amazon Web Services',
+//     regionId: 'us-east-2',
+//     regionName: 'US East (Ohio)',
+//   },
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: false,
+//     priority: 3,
+//     providerId: 'AWS',
+//     providerName: 'Amazon Web Services',
+//     regionId: 'us-west-2',
+//     regionName: 'US West (Oregon)',
+//   },
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: true,
+//     priority: 3,
+//     providerId: 'AWS',
+//     providerName: 'Amazon Web Services',
+//     regionId: 'us-west-2-2',
+//     regionName: 'US West (Oregon)',
+//   },
+// ]
+// const FAKE_AZURE: Cluster[] = [
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: true,
+//     priority: 1,
+//     providerId: 'Azure',
+//     providerName: 'Microsoft Azure',
+//     regionId: 'eastus',
+//     regionName: 'Virginia',
+//   },
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: true,
+//     priority: 2,
+//     providerId: 'Azure',
+//     providerName: 'Microsoft Azure',
+//     regionId: 'westeurope',
+//     regionName: 'Amsterdam',
+//   },
+// ]
+// const FAKE_GCP: Cluster[] = [
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: true,
+//     priority: 1,
+//     providerId: 'GCP',
+//     providerName: 'Google Cloud',
+//     regionId: 'eu-west1',
+//     regionName: 'Belgium',
+//   },
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: true,
+//     priority: 2,
+//     providerId: 'GCP',
+//     providerName: 'Google Cloud',
+//     regionId: 'us-central1',
+//     regionName: 'Iowa',
+//   },
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: true,
+//     priority: 98,
+//     providerId: 'GCP',
+//     providerName: 'Google Cloud',
+//     regionId: 'us-central1-2',
+//     regionName: 'Iowa 2',
+//   },
+//   {
+//     isBeta: false,
+//     isProviderAvailable: true,
+//     isRegionAvailable: true,
+//     priority: 99,
+//     providerId: 'GCP',
+//     providerName: 'Google Cloud',
+//     regionId: 'us-west1',
+//     regionName: 'Oregon',
+//   },
+// ]
+// const FAKE_CLUSTERS: Cluster[] = [...FAKE_AWS, ...FAKE_AZURE, ...FAKE_GCP]
+
```

Comment:
Maybe this is an oafish criticism of mine, but do we need a context here?  This is one overlay - couldn't we simply handle the states we need in the parent component?  It doesn't seem like we have enough components that prop drilling is going to be an issue.

Reply:
If we do need access to a persistent unified state, we might also used Redux here (which would give us the flexibility to access the state in other parts of the app, and would keep the cluster data alongside the other data retrieved from quartz).